dydx/dydxStateManager/dydxStateManager/AbacusState+Combine.swift

```diff
@@ -104,28 +104,20 @@ public final class AbacusState {
             .eraseToAnyPublisher()
     }
 
-    /**
-     Account balances (wallet balances)
-     **/
-    public enum NativeTokenDenom: String {
-        case usdc = "ibc/8E27BA2D5493AF5636760E354E46004562C46AB7EC0CC4C1CA14E9E20E2545B5"
-        case dydx = "dv4tnt"
-    }
-
-    public func accountBalance(of tokenDenom: NativeTokenDenom) -> AnyPublisher<Double?, Never> {
+    public func accountBalance(of tokenDenom: String) -> AnyPublisher<Double?, Never> {
```

Comment:
Nit: Can make the param optional String, so that the caller doesn't have to default to "".

Reply:
addressed 93c3720dd9b9b267ea23975cb87a0b6bb44ebbc3